### ISSUE TITLE
Suggestion: Move the buttons on Route Editor to the "Overflow" menu

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/OsmAnd#6532]
As shown below, the Route Editor has three buttons within its screen, which have no relationship with the POIs listed on the left. And yet they are placed exactly next to each POI, which creates the illusion that they are related to the POIs. 

Note that they take up precious space, and truncate the POI names (especially when we have 2 or more waypoints).  

![image](https://user-images.githubusercontent.com/9047168/52550335-b9482680-2dfd-11e9-8d12-36572f29b1a0.png)

In the screenshot, there are two waypoints: "Home" and "Near MS Ramaiah Road...".
See how awkward the combination looks.

**Desired:**
1. Move all buttons to the **Overflow** menu, and free the screen space.
2. List all waypoints properly, and separate them with a symbol.

**Comment by dmpr0** [comment]
Hello! Thank you for the feedback.
Thank you for using nightly builds, helping us to find bugs, but please don't forget that nightly build is not release version, the Directions screen is still in the development stage.

The buttons on the right side are partly related to the cells. On the first cell, we have the button to swap Start and Destination, the second cell has the button to open Waypoints screen, where you can delete/add/edit the full list of waypoints. The Add button maybe not so related, but it allows to add a waypoint in less tap. 
I don't want to hide all these buttons to the overflow menu. In the final stage, after opening these screen, titles will be faded, and only icons will stay on the screen.

**Comment by raindropsfromsky** [comment]
Yes, I am aware of the functions of those buttons (I am using the
development versions for several years now, and you can count me as per
user for most cases.)

Sometimes testers end up giving feedback on a post that is yet not settled.
IMHO the best way to avoid this is to declare a wiki where a feature
roadmap is declared, and as the features get matured, they are marked
appropriately, so that we can test them comprehensively.
*********
I do appreciate your point about not hiding the buttons, and making them
available upfront.

But in that case, can you make a two-column layout?

Also, the right column can be made very nature, by omitting the text labels
for the buttons. In fact, the *swap* button is already without a label. The
labels for the *Add *and* options *buttons can also be removed.

Without the labels, these buttons can be accommodated in a very narrow
column at right.

That way, you will have more space in the left column, to accommodate
long POI names.

**Comment by raindropsfromsky** [comment]
Sorry my last post was entered from a mobile phone, and contained errors. I have corrected them. 
I also elaborated the "developers' roadmap wiki" idea.

In fact, Blender is doing this for the last one year.

They have a weekly YouTube live telecast on their **Blender Today** channel. 
Pablo (the host) conducts an hour-long show in English.

He begins by describing each change in the GUI, and shows how that changes the user's workflow.
Once he covers all changes in details, he takes questions from viewers.
(Users can post question any time during the week at the [Blender Community portal](https://blender.community/c/).)

Immediately after the live show, he does it again in Spanish for "Blender Hoy" channel. 
There is a separate Spanish community page.

Worth emulating!
